R/scheduler.R

```diff
@@ -33,6 +33,10 @@ launch_code <- function(queue, workers, code) {
 
 resolve_worker <- function(id, graph, queue, workers) {
   if (future::resolved(workers[[id]])) {
+    # TODO: Process this value, it will contain
+    # graph updates
+    future::resolve(workers[[id]])
```

Comment:
I find it counterintuitive that we need to call `resolve()` on a future that is already resolved. @HenrikBengtsson, how does this work?

Reply:
I think this actually makes sense now. We are calling `resolve()` to assert that the future is resolved and then to the post-work in the outer future.